### ISSUE TITLE
Can't update cli caused by GitHub API limit.

### ISSUE THREAD

**Issue by dmnlk** [CircleCI-Public/circleci-cli#241]
tried update circleci-cli, but can't.
Error message is this.

```

[]:~/ circleci update 
Error: error finding latest release: GET https://api.github.com/repos/CircleCI-Public/circleci-cli/releases: 403 API rate limit exceeded for XXX.XXX.XXX.XXX. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) [rate reset in 27m17s]
```

This is GitHub API Limit, but if circleci cli can set GitHub personal token, I can avoid this error.
I want to can set GitHub token option like `--github-token`

**Comment by zzak** [comment]
Hello @dmnlk,

That is a correct assessment regarding the GitHub API rate limit and token.

You can set it via `GITHUB_TOKEN` environment variable, but we do not handle configuring your GitHub token in the CircleCI CLI.

**Comment by dmnlk** [comment]
@zzak 
thanks! I can updated!
is documented this behavior? if not , I will update README and send PullRequest.

**Comment by zzak** [comment]
@dmnlk We have a card in our backlog to document this behavior and make handling these errors nicer.

In the mean-time there's a much smaller task of adding a FAQ section to the README which includes this kind of troubleshooting. Does that sound like something you'd be interested in sending a PR for?

**Comment by dmnlk** [comment]
@zzak yes, I want to contribute this product. I'll send FAQ Pull Request asap!

**Comment by zzak** [comment]
This should be fixed by #267 and #268 where we now print a much more helpful message when this command fails.

Thanks again for your report!
